composer/datasets/synthetic.py

```diff
@@ -21,6 +29,88 @@ class SyntheticDataLabelType(StringEnum):
     CLASSIFICATION_ONE_HOT = "classification_one_hot"
 
 
+class SyntheticBertTokenizer(BertTokenizer):
+
+    def __init__(self, dataset, vocab_size=256):
+        try:
+            import tokenizers
+        except ImportError as e:
+            raise ImportError(
+                'Composer was installed without NLP support. To use NLP with Composer, run: `pip install mosaicml[nlp]`.'
+            ) from e
+
+        tokenizer = tokenizers.Tokenizer(tokenizers.models.WordPiece())
```

Comment:
is it possible to set the `normalizer`, `decoder`, etc. in the `Tokenizer` initialization function itself, rather than in subsequient lines?